cgi-bin/LJ/SynSuck.pm

```diff
@@ -61,6 +61,17 @@ sub delay {
     return undef;
 }
 
+sub max_size {
+    my ( $u ) = @_;  # optional user object for feed
+    my $max_size = $LJ::SYNSUCK_MAX_SIZE || 3000; # in kb
+
+    if ( $u && $u->has_priv( "siteadmin", "largefeedsize" ) ) {
+        $max_size = $LJ::SYNSUCK_LARGE_MAX_SIZE || 6000; # in kb
```

Comment:
If you feel like simplifying I'm pretty sure we can rip this out and just set the default to 6MB. This was written in a time when web servers have 4GB of RAM and such.